MapboxNavigation/Style.swift

```diff
@@ -209,6 +209,27 @@ public class ResumeButton: UIControl {
     }
 }
 
+/// :nodoc:
+@objc(MBDraggableView)
+open class StepListIndicatorView: UIView {
+    
+    // Workaround the fact that UIView properties are not marked with UI_APPEARANCE_SELECTOR
+    @objc dynamic open var gradientColors: [UIColor] = [.gray, .lightGray, .gray]
+    
+    override open func layoutSubviews() {
+        super.layoutSubviews()
+        applyGradient(colors: gradientColors)
+        layer.cornerRadius = bounds.midY
+        layer.masksToBounds = true
+        layer.opacity = 0.25
+        
+        let blurEffect = UIBlurEffect(style: .extraLight)
+        let blurredEffectView = UIVisualEffectView(effect: blurEffect)
```

Comment:
If something would trigger a re-layout, such as changing the font size in the accessibility inspector, change style, the `UIVisualEffectView` gets duplicated. We should keep a reference to the `blurredEffectView` and only add it if it's not already added.

![screenshot 2018-03-25 23 23 26](https://user-images.githubusercontent.com/764476/37880130-0de7e0c2-3084-11e8-961e-64aa69e24bd2.png)